modules/tutorials/pages/java-tutorial/create-records.adoc

```diff
@@ -0,0 +1,79 @@
+= Create Student and Course Records
+:description: Learn how to use the SDK to create student and course records.
+:page-pagination: full
+:page-topic-type: tutorial
+
+[abstract]
+{description}
+
+== Create a Student Record
+
+After connecting to the cluster, you can create a student record in the form of a JSON document inserted into the `student-record-collection`.
+
+To create a student record:
+
+. In your `java` directory, create a new file called `InsertStudent.java`.
+. Paste the following code block into your `InsertStudent.java` file:
++
+[source, java]
+----
+include::example$InsertStudent.java[]
+----
++
+. Open a terminal window and navigate to your `student` directory.
+. Run the command `mvn install` to pull in all the dependencies and rebuild your application.
+. Run the following command to insert the student record into the collection:
++
+[source, sh]
+----
+mvn exec:java -Dexec.mainClass="InsertStudent" -Dexec.cleanupDaemonThreads=false
+----
++
+. From the Capella UI, go to your student cluster and check the `student-record-collection` for the new student record you just added:
+.. Go to menu:Data Tools[Documents].
+.. Under *Get documents from*, select `student-bucket`, `art-school-scope`, and `student-record-collection`.
+.. Click btn:[Get Documents].
++
+image::new-student-record.png[alt="Student added to the student-record-collection in the cluster"]
+
+
+== Create Course Records
+
+Creating course records is similar to creating student records.
+To create course records:
+
+. In your `java` directory, create a new file called `InsertCourses.java`.
+. Paste the following code block into your `InsertCourses.java` file:
++
+[source, java]
+----
+include::example$InsertCourses.java[]
+----
++
+. Open a terminal window and navigate to your `student` directory.
```

Comment:
Again, tell them explicitly to fill in the username, password, and connection string.

```suggestion
. In the `InsertCourses.java` file, replace the `<<connection-string>>`, `<<username>>`, and `<<password>>` placeholders with the connection string, username, and password that you noted when configuring the cluster connection.
. Open a terminal window and navigate to your `student` directory.
```